[2024-06-03T11:35:07.384+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T11:35:07.408+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: city_data_workflow_DAG_final.group_tasks.is_weather_api_ready manual__2024-06-03T11:05:27.638846+00:00 [queued]>
[2024-06-03T11:35:07.419+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: city_data_workflow_DAG_final.group_tasks.is_weather_api_ready manual__2024-06-03T11:05:27.638846+00:00 [queued]>
[2024-06-03T11:35:07.420+0000] {taskinstance.py:2306} INFO - Starting attempt 6 of 11
[2024-06-03T11:35:07.445+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): group_tasks.is_weather_api_ready> on 2024-06-03 11:05:27.638846+00:00
[2024-06-03T11:35:07.455+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1763) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T11:35:07.456+0000] {standard_task_runner.py:63} INFO - Started process 1768 to run task
[2024-06-03T11:35:07.456+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'city_data_workflow_DAG_final', 'group_tasks.is_weather_api_ready', 'manual__2024-06-03T11:05:27.638846+00:00', '--job-id', '1091', '--raw', '--subdir', 'DAGS_FOLDER/tasks_flow.py', '--cfg-path', '/tmp/tmphrshj37d']
[2024-06-03T11:35:07.458+0000] {standard_task_runner.py:91} INFO - Job 1091: Subtask group_tasks.is_weather_api_ready
[2024-06-03T11:35:07.535+0000] {task_command.py:426} INFO - Running <TaskInstance: city_data_workflow_DAG_final.group_tasks.is_weather_api_ready manual__2024-06-03T11:05:27.638846+00:00 [running]> on host cd9caf1c47f0
[2024-06-03T11:35:07.713+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='city_data_workflow_DAG_final' AIRFLOW_CTX_TASK_ID='group_tasks.is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T11:05:27.638846+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T11:05:27.638846+00:00'
[2024-06-03T11:35:07.715+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T11:35:07.733+0000] {http.py:137} INFO - Poking: /data/2.5/weather?lat=10.82302&lon=106.62965&appid=cb9774016f254a4cb1aa6516bf401afa
[2024-06-03T11:35:07.750+0000] {base.py:84} INFO - Using connection ID 'weather_map_api_connection' for task execution.
[2024-06-03T11:35:08.678+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T11:35:08.679+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T11:35:08.698+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=city_data_workflow_DAG_final, task_id=group_tasks.is_weather_api_ready, run_id=manual__2024-06-03T11:05:27.638846+00:00, execution_date=20240603T110527, start_date=20240603T113507, end_date=20240603T113508
[2024-06-03T11:35:08.757+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T11:35:08.773+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
